src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
```

Comment:
- [ ] Please make sure you're importing the `categories` file from `./categories/categories` 🙏